share/oshu!.desktop

```diff
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Type=Application
+Name=oshu!
+GenericName=Rhythm Game
+GenericName[fr]=Jeu de Rythme
+Comment=Fast osu! port
+Comment[fr]=Adaptation rapide de osu!
+Exec=oshu %F
+MimeType=text/x-osu
+Terminal=true
+Categories=Game;X-Music;X-Rhythm
```

Comment:
The bang in the filename may cause trouble for some systems. Calling it oshu without the bang is consistent with the executable file, so I'd rather play it safe.